schema.sql

```diff
@@ -1,14 +1,52 @@
 /* Database schema to keep the structure of entire database. */
-
 CREATE DATABASE vet_clinic;
-
 CREATE TABLE animals(
-id INT PRIMARY KEY,
-name VARCHAR,
-date_of_birth DATE,
-escape_attempts INT,
-neutered BOOLEAN,
-weitght_kg DECIMAL);
-
+  id INT,
+  name TEXT,
+  date_of_birth DATE, 
+  escape_attemps INT,
+  neutered BOOLEAN,    
+  weight_kg DEC,
+  species_id INT,
+  owner_id INT,
+  PRIMARY KEY(id),
+  FOREIGN KEY(species_id) REFERENCES species(id),
+  ADD FOREIGN KEY(owner_id) REFERENCES species(id)
+);
+ALTER TABLE animals 
+ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY;
 ALTER TABLE animals
-ADD species TEXT ;
+DROP COLUMN species;
+ALTER TABLE animals ADD species_id INT;
+ALTER TABLE animals ADD FOREIGN KEY(species_id) REFERENCES species(id);
+ALTER TABLE animals ADD owner_id INT;
+ALTER TABLE animals ADD FOREIGN KEY(owner_id) REFERENCES species(id);
+CREATE TABLE owners(
+ id INT GENERATED ALWAYS AS IDENTITY,
+ full_name TEXT,
+ age INT,
+ PRIMARY KEY(id)
+);
+CREATE TABLE species(
+ id GENERATED ALWAYS AS IDENTITY,
+ name TEXT,
+ PRIMARY KEY(id)
+);
+
+CREATE TABLE vets(
+ id INT GENERATED ALWAYS AS IDENTIY,
```

Comment:
### Optional
 - [ ] It would be better if we set the ID as a primary key, so that we can uniquely identify each fields in the database